docs/examples/plot_transposition_gain.py

```diff
@@ -0,0 +1,119 @@
+"""
+Modeling Transposition Gain
+===========================
+
+Calculating the gain in insolation of a tilted module over a flat module.
+"""
+
+# %%
+# This example shows how to evaluate the transposition gain of a racking
+# strategy.  The transposition gain is the additional insolation collected
+# by orienting at a tilt instead of flat; using PV modeling lingo, it's the
+# increase in POA (plane of array) insolation over GHI (global horizontal
+# irradiance) insolation.
+#
+# This example uses a TMY dataset and the
+# :py:meth:`pvlib.irradiance.get_total_irradiance` function to transpose
+# irradiance components to POA irradiance for various fixed tilts.  It also
+# models a single-axis tracking system for comparison. The monthly POA
+# insolation is calculated for each strategy to show how orientation affects
+# seasonal irradiance collection.
+
+import pvlib
+from pvlib import location
+from pvlib import irradiance
+from pvlib import tracking
+from pvlib.iotools import read_tmy3
+import pandas as pd
+from matplotlib import pyplot as plt
+import pathlib
+
+# get full path to the data directory
+DATA_DIR = pathlib.Path(pvlib.__file__).parent / 'data'
+
+# get TMY3 dataset
+tmy, metadata = read_tmy3(DATA_DIR / '723170TYA.CSV', coerce_year=1990)
+# trim off the last row to make resampling easier
```

Comment:
?

Reply:
The last row of the raw TMY dataframe is `1991-01-01 00:00:00-05:00`, so resampling to monthly creates a spurious `1991-01` value.  In hindsight that explanatory comment could certainly be improved.  Or is there a cleaner way to handle the "annual roll over" from right-labeling?

Reply:
LGTM with the edits